### PR TITLE
Merge changes from hotfix_5.0.2 into release

### PR DESCRIPTION
**Pull request by b-cooper**
## Overview

PR to update the release branch with the changes made and QA'd as a part of the 5.0.2 hotfix release.

## Changelog

- **api:** lowercase filename before checking extension in ProtocolReader ([#9609])
- **api, app:** display labware label in labware setup ([#9587](https://github.com/Opentrons/opentrons/issues/9587)) ([7680d92])
- **app:** do not pass `directionControlButtonColor` into DOM ([#9615])
- **app:** show labware def display name in lpc summary screen, fix nested lw render condition ([#9614])
- **app:** surface run record creation errors to users on upload page ([#9597])
- send custom labware definitions to the server during LPC setup ([#9588])

## Review requests

Standard hotfix release QA

## Risk assessment

Very low

### CODE REVIEW

**Review by shlokamin — APPROVED** [review]
changes look good to me!

**Review by nusrat813 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [5.0.2](https://github.com/Opentrons/opentrons/compare/v5.0.1...v5.0.2) (2022-03-03)
+
+
+### Bug Fixes
+
+* **api:** lowercase filename before checking extension in ProtocolReader ([#9609](https://github.com/Opentrons/opentrons/issues/9609)) ([474c448](https://github.com/Opentrons/opentrons/commit/474c44860ac8b29c621b3a704e826688aa57ca59))
+* **api, app:** display labware label ([#9587](https://github.com/Opentrons/opentrons/issues/9587)) ([7680d92](https://github.com/Opentrons/opentrons/commit/7680d92d4a966d641e8da558514a153e06946bce)), closes [#9105](https://github.com/Opentrons/opentrons/issues/9105) [#9088](https://github.com/Opentrons/opentrons/issues/9088)
```
Comment (sanni-t) [inline CHANGELOG.md:12]: Do we usually include the 'closes' tickets in the changelog?

**Review by sanni-t — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (b-cooper) [reply CHANGELOG.md:12]: Yup, they get included as a part of the autogenerated commitizen changelog

**Review by mcous — APPROVED** [review]

**Pull request merged**